run.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+# Start the proxy
+./cloud_sql_proxy -instances=$CLOUDSQL_INSTANCE=tcp:5432 -credential_file=$CLOUDSQL_CREDENTIALS &
+
+# wait for the proxy to spin up
+sleep 10
```

Comment:
Consider [waiting till the port is bound](https://stackoverflow.com/questions/27599839/how-to-wait-for-an-open-port-with-netcat) since different machines will take different amounts of time here.

Reply:
Good call, just updated it

Reply:
sick!